openid-4-verifiable-presentations-1_0.md

```diff
@@ -867,15 +868,36 @@ In detail, the array is processed by the Wallet from left to right as follows:
        currently selected element(s) is not an array, abort processing and
        return an error. If the index does not exist in a selected array, remove
        that array from the selection.
-  3. If the set of elements currently selected is empty, abort processing and
-     return an error.
+3. If the set of elements currently selected is empty, abort processing and
+   return an error.
+
+The result of the processing is the set of selected JSON elements.
+
+## Semantics for ISO mdoc-based credentials
```

Comment:
Why is this section here and not in the mdoc credential format appendix? It's explicitly for ISO mdoc, so I don't understand why it should be here.

Reply:
i think for the symmetry with JSON-based credentials section. will merge as-is since there are enough approvals and no other blockers. we can fix this later if necessary, as we did with the mdoc section in the very first DCQL PR.